e3db/tests/test_notes.py

```diff
@@ -120,22 +152,81 @@ def test_encrypt_field(self):
   def test_encrypt_valid_note_suceeds(self):
     '''Asserts that a note with the correct encrypted data is returned'''
     
-    note_options_A = note_options # to be made test specific later
-    
+    note_options_A = NoteOptions(
+      note_writer_client_id=self.client1.client_id,
+      max_views=3,
+      id_string=f"globalNoteName-${uuid4()}",
+      expiration='0001-01-01T00:00:00Z',
+      expires=False,
+      type='',
+      plain={},
+      file_meta={}
+    )
     # NOTE: KEYS ARE BASE64URLENCODED string of bytes
-
-    print(f'private signing key: {writer_private_signing_key_A}')
-    print(f'length of private key base64 encoded: {len(writer_private_signing_key_A)}')
-
     encrypted_note_A = e3db.Client.create_encrypted_note(unencrypted_data_A, 
                              reader_public_key_A,
                              reader_public_signing_key_A, 
                              writer_key_pair_A,
                              writer_signing_key_pair_A,
                              note_options_A)
 
-    print(f'reader private key: {reader_private_key_A}')
-    print(f'writer public signing key: {writer_public_signing_key_A}, writer private signing key: {writer_private_signing_key_A}')
     decrypted_note_A = e3db.Client.decrypt_note(encrypted_note_A, reader_private_key_A)
     assert(decrypted_note_A.data == unencrypted_data_A)
+    
+  def test_write_note(self):
```

Comment:
Question about code organization: Does this and `test_write_and_read_a_note` belong in `test_integration.py`? Either it makes more sense to put there, or the read note test should be moved to here?

Reply:
The other tests in `test_notes.py` remind me of `test_tsv1_auth.py`, in which they are testing individual components of notes but not actually making a connection to a server

Reply:
I found it more convenient to have all the tests for one subject in one place. Do you have a strong preference? If not I will leave them here and we can refactor later as needed.

Reply:
No strong preference, so we can come back to this later if anybody else has an opinion